packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java

```diff
@@ -517,6 +520,35 @@ public void onResponse(Call call, Response response) throws IOException {
     }
   }
 
+  @ExpoMethod
+  public void getContentUriAsync(String uri, Promise promise){
+    try{
+      final Uri fileUri = Uri.parse(uri);
+      ensurePermission(fileUri, Permission.WRITE);
+      ensurePermission(fileUri, Permission.READ);
+      checkIfFileDirExists(fileUri);
+      if("file".equals(fileUri.getScheme())){
+        File file = uriToFile(fileUri);
+        Bundle result = new Bundle();
+        result.putString("uri", cUriFromFile(file).toString());
+        promise.resolve(result);
+      }
```

Comment:
add an else clause here that rejects the promise. otherwise we could potentially get into a state where the promise hangs indefinitely and is never resolved or rejected, which we always want to avoid.

Reply:
Sure!